hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientMessage.java

```diff
@@ -395,7 +395,7 @@ public Frame peekNext() {
     @SuppressWarnings("checkstyle:VisibilityModifier")
     public static class Frame {
         public final byte[] content;
-        //begin-fragment end-fragment final begin-data-structure end-data-structure is-null is-event 9reserverd
+        //begin-fragment end-fragment final begin-data-structure end-data-structure is-null is-event 9reserved
```

Comment:
The tool I'm using doesn't see numbers. I do, but I have no idea what this is trying to say.

Reply:
I'd say it's "9 reserved bits", but didn't dig very deep. I think fixing the typo at least is fine.